apps/mobile/src/libs/styles/global-styles/paddings.ts

```diff
@@ -0,0 +1,68 @@
+import { SPACING } from "~/libs/styles/global-styles/spacing";
+
+const PADDINGS = {
```

Comment:
```suggestion
const Padding = {
```

Same for others

Reply:
Also, do I need to change uppercase to capital case?